content/_data/events/st.petersburg_jam_feb2017.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "St.Petersburg JAM"
```

Comment:
Maybe it worth to mention it is St. Petersburg in Russia, not the one in Florida